### PR TITLE
Fix static file handling

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

**Comment by akatsoulas** [comment]
r+

File: cardano/settings.py [hunk 1]
```diff
@@ -73,6 +73,7 @@
 STATIC_HOST = config('STATIC_HOST', default='')
 STATIC_ROOT = config('STATIC_ROOT', default=os.path.join(BASE_DIR, 'static'))
 STATIC_URL = config('STATIC_URL', STATIC_HOST + '/static/')
+STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
```
Comment (comzeradd) [inline cardano/settings.py]: I think this should be:

```
'whitenoise.django.GzipManifestStaticFilesStorage'
```

At least for me the above fails locally

File: cardano/settings.py [hunk 2]
```diff
@@ -35,6 +34,8 @@
 )
 
 MIDDLEWARE_CLASSES = (
+    'django.middleware.security.SecurityMiddleware',
+    'whitenoise.middleware.WhiteNoiseMiddleware',
```
Comment (comzeradd) [inline cardano/settings.py]: This also fails for me locally. See below.

**Comment by johngian** [comment]
@comzeradd PR updated

**Comment by comzeradd** [comment]
Tested locally. r+

**Pull request merged**